pegasus/sites.v3/code.org/public/hourofcode/overview.haml

```diff
@@ -17,7 +17,6 @@ social:
 %link{href: "/css/student.css", rel: "stylesheet"}
 %link{href: "/css/tools.css", rel: "stylesheet"}
 %link{href: "/shared/css/course-blocks.css", rel: "stylesheet", type: "text/css"}
-%script{type: "text/javascript", src: "/js/hoc-overview.js"}
```

Comment:
There was a strong movement to get JS out of HAML and into its own files, so I think some folks will see this as a step backwards...

Reply:
i was struggling with that too. it felt weird to have a .js file with one line of javascript, but maybe it's better to just be consistent. i'll move this back into its own file